src/App.tsx

```diff
@@ -1,24 +1,242 @@
-/* eslint-disable max-len */
```

Comment:
Please remove the unnecessary eslint-disable comments (max-len and jsx-a11y/control-has-associated-label) as they are not needed in this file.